Add support for new branch triggers from Bitbucket Server

copini opened the pull request:
This pull request fixes a bug causing newly pushed branches on a Bitbucket Server repository not to trigger a build in Bitrise. See issue #69.

Fixes #69

viktorbenei commented on service/hook/bitbucketserver/bitbucketserver_test.go:
```diff
@@ -51,7 +51,7 @@ const (
       "fromHash":"from-hash-1",
       "toHash":"to-hash-1",
-      "type":"UPDATE"
+      "type":"ADD"
```
Please make this an additional test case, do not modify the previous one(s)

copini commented:
You're totally right, I've reverted those changes and added a specific test case for pushing a new branch. Including the "000.." `fromHash` that the actual webhook data for these kind of events has.

Please let me know if this is sufficient!
